base_layer/wallet/src/output_manager_service/storage/database.rs

```diff
@@ -700,10 +700,7 @@ where T: OutputManagerBackend + 'static
         let db_clone = self.db.clone();
         tokio::task::spawn_blocking(move || {
             match db_clone.write(WriteOperation::Remove(DbKey::AnyOutputByCommitment(commitment.clone()))) {
-                Ok(None) => log_error(
-                    DbKey::AnyOutputByCommitment(commitment.clone()),
-                    OutputManagerStorageError::ValueNotFound,
-                ),
+                Ok(None) => Ok(()),
```

Comment:
Why remove the logging here?
If its confusing or provides false errors, perhaps reword the log.

Reply:
Logging was removed here because it is not an error. It is a valid condition that happens a lot and is not worth logging. @philipr-za, I know we discussed this; your opinion, please?

Reply:
This is expected to happen a lot. Not finding the the output is not an error and I do not think it is notable in the logs. I would argue that if code is using this and not finding the output is a concern to that code that it should log the problem up at that layer.